types/w3c-web-serial/index.d.ts

```diff
@@ -53,6 +53,7 @@ declare class SerialPort extends EventTarget {
     getSignals(): Promise<SerialInputSignals>;
     getInfo(): SerialPortInfo;
     close(): Promise<void>;
+    forget?(): Promise<void>;
```

Comment:
This method is required by the specification and so shouldn't be marked optional.

Reply:
ok